neo4j/_async/work/session.py

```diff
@@ -278,9 +280,15 @@ async def job(tx, **job_kwargs):
                 return QueryResult([], summary)
             return await tx.query(query, parameters, **job_kwargs)
 
-        return await self.execute(job, **kwargs)
+        return await self.execute(
+            job,
+            cluster_member_access=cluster_member_access,
+            **kwargs
+        )
 
-    async def execute(self, transaction_function, *args, **kwargs):
+    async def execute(self, transaction_function, *args,
+                      cluster_member_access=CLUSTER_AUTO_ACCESS,
+                      **kwargs):
```

Comment:
1. about the code-style. This is fine, but when run through unasync will look like this
```python
    def execute(self, transaction_function, *args,
                      cluster_member_access=CLUSTER_AUTO_ACCESS,
                      **kwargs):
```
:nauseated_face: 

This can be fixed like so
```suggestion
    async def execute(
        self, transaction_function, *args, 
        cluster_member_access=CLUSTER_AUTO_ACCESS, **kwargs
    ):
```

There are other places like this. Basically whenever you have an async function, with multiple lines of arguments, put the first argument on a new line. Note that this is not general Python style-guide, but just necessary to make the output of unasync look neat.